internal/brokers/examplebroker/dbus.go

```diff
@@ -12,8 +12,9 @@ import (
 )
 
 const (
-	dbusObjectPath = "/com/ubuntu/auth/ExampleBroker"
-	dbusInterface  = "com.ubuntu.auth.ExampleBroker"
+	dbusName       = "com.ubuntu.authd.ExampleBroker"
+	dbusObjectPath = "/com/ubuntu/authd/ExampleBroker"
+	dbusInterface  = "com.ubuntu.authd.ExampleBroker"
```

Comment:
I was unsure whether it would be a good idea. I added the dbusName constant to keep it more explicit, but we definitely can use the `dbusInterface` as the name.